umpleonline/ump/manualexamples/E074DuplicateParallelStateMachineNameInParallelRegion1.ump

```diff
@@ -0,0 +1,28 @@
+// The following example generates the error
```

Comment:
With the current version of Umple, the following enums are generated when this file is compiled into Java:

```
//X State Machines
  public enum Sm { s0, s1 }
  public enum SmT1 { Null, t1 }
  public enum SmT2 { Null, t2, t3 }
  public enum SmT1 { Null, t1, t4 }
  public enum SmT2 { Null, t2, t5 }
  private Sm sm;
  private SmT1 smT1;
  private SmT2 smT2;
  private SmT1 smT1;
  private SmT2 smT2;
```